includes/abilities/class-wp-core-abilities.php

```diff
@@ -0,0 +1,288 @@
+<?php
+/**
+ * Core Abilities registration.
+ *
+ * @package WordPress
+ * @subpackage Abilities_API
+ * @since 0.3.0
+ */
+
+declare( strict_types = 1 );
+
+/**
+ * Registers the default core abilities that ship with the Abilities API.
+ *
+ * @since 0.3.0
+ */
+// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedClassFound -- Core class intended for WordPress core.
+class WP_Core_Abilities {
```

Comment:
Keeps iteration future-compat

```suggestion
final class WP_Core_Abilities {
```